modules/reindex/src/main/plugin-metadata/entitlement-policy.yaml

```diff
@@ -1,7 +1,20 @@
 ALL-UNNAMED:
   - manage_threads
   - outbound_network
+  - create_class_loader # needed for Painless to generate runtime classes
   - files:
-      - relative_path: ""
+      - relative_path_setting: "reindex.ssl.certificate"
```

Comment:
@rjernst I have no idea if I'm using this correctly, as this is the first time we use it I think.
Will this work with exclusive access?
Can I use a glob (reindex.ssl.*) instead?
`certificate_authorities` is a list; will that work with your file settings implementation?

Reply:
Yes, I think so!

Reply:
> Also note the properties are changing per our earlier discussion, see https://github.com/elastic/elasticsearch/pull/123649.

Thanks! So `reindex.ssl.*` should work? I'll try it/look at logs/debug what it resolves to. Or maybe I'll write a test for it.
I'll update this to use the new properties after that is merged.